### PR TITLE
Add bpruitt-goddard Ergodox Layout

### PR DESCRIPTION
**Pull request by bpruitt-goddard**
Ergodox EZ layout with Included KLE image and link to provide others with a quick idea of what features the layout provides.

### CODE REVIEW

File: keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+//Move mac desktop spaces
+#define MAC_L     LGUI(LSFT(KC_UP)) // cmd + shift + up
+#define MAC_R     LGUI(LSFT(KC_DOWN)) // cmd + shift + down
+
+// This file must be included after DYNAMIC_MACRO_RANGE is defined...
+#include "dynamic_macro.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  // Layer 0: basic keys.
+  [BASE] = KEYMAP(
```
Comment (drashna) [inline keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c]: Could you change the `KEYMAP`s to `LAYOUT_ergodox`?

Comment (bpruitt-goddard) [reply keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c]: I hadn't seen that! I updated my `KEYMAP`s to be the new pretty format (see current PR version) ✨.

File: keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+void led_2_off(void) {
+  ergodox_right_led_2_off();
+}
+
+void led_3_off(void) {
+  ergodox_right_led_3_off();
+}
+
+uint32_t layer_state_set_user(uint32_t state) {
+  current_layer_state = state;
```
Comment (drashna) [inline keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c]: This isn't actually needed.  You can use `layer_state`: https://github.com/qmk/qmk_firmware/blob/7c9d5ace143d3cc6d767a354acde814926d566fd/tmk_core/common/action_layer.c#L119

Comment (drashna) [reply keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c]: But this is minor, and fine.

**Comment by drashna** [comment]
Yeah, the pretty layout is much nicer, visually!   I use it personally, and don't need ASCII art or images, as much. :)

And thanks!

**Pull request merged**